src/mscorlib/shared/System/Diagnostics/StackTraceHiddenAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics
+{
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Constructor | AttributeTargets.Struct, Inherited = false)]
```

Comment:
Nothing is looking for this attribute on properties today - it may be better to omit `AttributeTargets.Property`.

Reply:
removed